### ISSUE TITLE
Test failure: System.Net.Http.Functional.Tests.ImpersonatedAuthTests.DefaultHandler_ImpersonatedUser_Success

### ISSUE THREAD

**Issue by rzikm** [dotnet/runtime#65712]
```
System.Net.Http.Functional.Tests.ImpersonatedAuthTests.DefaultHandler_ImpersonatedUser_Success(useNtlm: True) [FAIL]
    System.Net.Http.Functional.Tests.ImpersonatedAuthTests.DefaultHandler_ImpersonatedUser_Success(useNtlm: False) [FAIL]
      System.AggregateException : One or more errors occurred. (Unable to find an entry point named 'LogonUser' in DLL 'advapi32.dll'.) (The following constructor parameters did not have matching fixture data: WindowsIdentityFixture windowsIdentityFixture)
      ---- System.EntryPointNotFoundException : Unable to find an entry point named 'LogonUser' in DLL 'advapi32.dll'.
      ---- The following constructor parameters did not have matching fixture data: WindowsIdentityFixture windowsIdentityFixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.EntryPointNotFoundException) -----
           at System.WindowsTestAccount.<LogonUser>g____PInvoke__|10_0(UInt16* userName, UInt16* domain, UInt16* password, Int32 logonType, Int32 logonProvider, IntPtr* safeAccessTokenHandle)
        /_/src/libraries/Common/tests/TestUtilities/Microsoft.Interop.DllImportGenerator/Microsoft.Interop.DllImportGenerator/GeneratedDllImports.g.cs(275,0): at System.WindowsTestAccount.LogonUser(String userName, String domain, String password, Int32 logonType, Int32 logonProvider, SafeAccessTokenHandle& safeAccessTokenHandle)
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(91,0): at System.WindowsTestAccount.CreateUser()
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(44,0): at System.WindowsTestAccount..ctor(String userName)
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(23,0): at System.WindowsIdentityFixture..ctor()
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
      System.AggregateException : One or more errors occurred. (Unable to find an entry point named 'LogonUser' in DLL 'advapi32.dll'.) (The following constructor parameters did not have matching fixture data: WindowsIdentityFixture windowsIdentityFixture)
      ---- System.EntryPointNotFoundException : Unable to find an entry point named 'LogonUser' in DLL 'advapi32.dll'.
      ---- The following constructor parameters did not have matching fixture data: WindowsIdentityFixture windowsIdentityFixture
      Stack Trace:
        
        ----- Inner Stack Trace #1 (System.EntryPointNotFoundException) -----
           at System.WindowsTestAccount.<LogonUser>g____PInvoke__|10_0(UInt16* userName, UInt16* domain, UInt16* password, Int32 logonType, Int32 logonProvider, IntPtr* safeAccessTokenHandle)
        /_/src/libraries/Common/tests/TestUtilities/Microsoft.Interop.DllImportGenerator/Microsoft.Interop.DllImportGenerator/GeneratedDllImports.g.cs(275,0): at System.WindowsTestAccount.LogonUser(String userName, String domain, String password, Int32 logonType, Int32 logonProvider, SafeAccessTokenHandle& safeAccessTokenHandle)
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(91,0): at System.WindowsTestAccount.CreateUser()
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(44,0): at System.WindowsTestAccount..ctor(String userName)
        /_/src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs(23,0): at System.WindowsIdentityFixture..ctor()
        ----- Inner Stack Trace #2 (Xunit.Sdk.TestClassException) -----
        
```

All versions of windows. Started on 2022-02-19.

**Comment by ManickaP** [comment]
main or particular PR?

**Comment by karelz** [comment]
Triage: 70 failures on main in last 3 days. We should address it.
@rzikm please check if it fails on retries. If it does, we should disable the test ASAP.

**Comment by rzikm** [comment]
The one failure I was able to track down was repeated: https://dev.azure.com/dnceng/public/_build/results?buildId=1622880&view=ms.vss-test-web.build-test-results-tab&runId=45099776&resultId=104268&paneView=dotnet-dnceng.dnceng-build-release-tasks.helix-test-information-tab

**Comment by jkotas** [comment]
Duplicate of https://github.com/dotnet/runtime/issues/65648. Fixed by #65745
